kairon/api/app/routers/bot/bot.py

```
@@ -708,6 +708,23 @@ async def fetch_logs(
     return Response(data={"logs": logs, "total": row_cnt})
 
 
+@router.get("/logs/{log_type}/search", response_model=Response)
+async def search_logs(
+    log_type: LogTypeEnum,
+    key: List[str] = Query([]),
```

Ruff (0.12.2)

Do not perform function call `Query` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)